src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -228,9 +270,20 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           </Button>
         </div>
       )}
-      <ul className="NavBarSchedule__weekdays">{weekdays}</ul>
 
-      <Schedule isLoading={isLoadingSchedule} {...schedule} />
+      {/* TODO: refactor this to be cleaner + follow our standards better */}
+      {renderedScheduleDayTabs.length > 0 ? (
+        <>
+          <ul className="NavBarSchedule__weekdays">
+            {renderedScheduleDayTabs}
+          </ul>
+
+          <Schedule isLoading={isLoadingSchedule} {...schedule} />
+        </>
+      ) : (
+        // TODO: Improve how this displays
+        <div>Scheduled events have ended</div>
+      )}
```

Comment:
[TODO] refactor this to be cleaner + follow our standards better + improve how "Scheduled events have ended" displays